container/deps/vllm/install_vllm.sh

```
@@ -275,4 +275,35 @@ if [ "$DEVICE" = "cuda" ]; then
+# LoRA, etc.) automatically in every vLLM worker process -- including spawned
+# subprocesses. Required for prime-rl / Dynamo RL training integration.
+#
+# Override at build time:  --build-arg PRIME_RL_REF=v0.5.1.dev101
+# --no-deps: prime-rl's full dep tree includes trainer + wandb; Dynamo only
+#            needs the inference-side plugin and worker-extension classes.
+# Python version: prime-rl pins requires-python = "~=3.12.0"; Dynamo containers
+#            are Python 3.12, so no version override is needed. For 3.11 local
+#            dev venvs use the regular pip (not uv) with --ignore-requires-python.
+# ---------------------------------------------------------------------------
+PRIME_RL_REF="${PRIME_RL_REF:-v0.5.1.dev101}"
+echo "\n=== Installing prime-rl vLLM plugin (ref=${PRIME_RL_REF}) ==="
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)